dokan/dokan.h

```diff
@@ -610,20 +627,17 @@ typedef struct _DOKAN_OPERATIONS {
   /**
   * \brief SetFileSecurity Dokan API callback
   *
-  * Set ACL information on the requested file.
-  *
-  * see Win32 API SetFileSecurity
-  * https://msdn.microsoft.com/en-us/library/windows/desktop/aa379577(v=vs.85).aspx
-  *
-  * Suported since 0.6.0. You must specify the version at
-  * DOKAN_OPTIONS.Version.
+  * Sets the security of a file or directory object.
   *
+  * \since Supported since version 0.6.0. You must specify the version in \ref DOKAN_OPTIONS#Version.
   * \param FileName File path requested by the Kernel on the FileSystem.
-  * \param SecurityInformation Structure that identifies the contents of the security descriptor pointed by SecurityDescriptor param.
-  * \param SecurityDescriptor A pointer to a SECURITY_DESCRIPTOR structure.
+  * \param SecurityInformation Structure that identifies the contents of the security descriptor pointed by \a SecurityDescriptor param.
+  * \param SecurityDescriptor A pointer to a \ref SECURITY_DESCRIPTOR structure.
```

Comment:
No ref possible here. SECURITY_DESCRIPTOR is not define in dokan but windows headers.